src/main/java/in/org/projecteka/hiu/valuseset/ValueSetResource.java

```diff
@@ -0,0 +1,27 @@
+package in.org.projecteka.hiu.valuseset;
+
+import org.apache.commons.io.IOUtils;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Component;
+
+import java.nio.charset.Charset;
+
+@Component
+public class ValueSetResource implements InitializingBean {
```

Comment:
The ConceptValidator class already does the loading. Maybe we can refactor the ConceptValidator to use this class instead?

Reply:
Sure!